lib/autocomplete/gocodeprovider.js

```diff
@@ -61,7 +61,6 @@ class GocodeProvider {
     this.subscriptions = null
     this.goconfig = null
     this.subscribers = null
-    this.selector = null
```

Comment:
See the commit msg. The specs were printing a warning about our provider not being valid.